db/database.go

```diff
@@ -178,8 +178,8 @@ type ScopeOptions struct {
 }
 
 type CollectionOptions struct {
-	Sync *string // Collection sync function
-
+	Sync          *string        // Collection sync function
+	ImportOptions *ImportOptions // Collection import filter and options
```

Comment:
I think it makes sense to just define this as the import filter (*string), since ImportOptions.ImportPartitions and BackupOldRev are database-wide settings.

Reply:
Fixed. It's the compiled string here I think.